pkg/assets/config.go

```diff
@@ -0,0 +1,67 @@
+package assets
+
+type Config struct {
+	BootstrapKubeAPIServerConfig
+	KubeAPIServerConfig
+
+	// ConfigDir is the directory where the "config.json" must be located
+	ConfigDir string
+
+	// DataDir is the /var/lib/origin directory
+	// TODO: Do we need this?
+	DataDir string
+
+	// CloudProviderDir is the directory with cloud provider specific data
+	CloudProviderDir string
+}
+
+type KubeAPIServerConfig struct {
+	// Namespace is the target namespace for the bootstrap kubeapi server to be created
+	Namespace string
+
+	// Image is the pull spec of the image to use for the api server.
+	Image string
+
+	// ImagePullPolicy specifies the image pull policy to use for the images.
+	ImagePullPolicy string
+
+	Secrets    KubeAPIServerSecretsConfig
+	ConfigMaps KubeAPIServerConfigMapsConfig
+}
+
+type KubeAPIServerSecretsConfig struct {
+	Namespace string
+
+	AggregatorClientCertCrt string
```

Comment:
these are what? base64?

Reply:
Yes, these are filled with base64 and injected to manifests

Reply:
Base64 postfix or prefix would make it clear